packages/office-ui-fabric-react/src/components/SwatchColorPicker/index.ts

```diff
@@ -1,2 +1,3 @@
 export * from './SwatchColorPicker';
+export * from './SwatchColorPicker.base';
 export * from './SwatchColorPicker.types';
```

Comment:
I think you moved an interface to another types files (IColorPickerCellProps or something.) can you make sure it's getting exported as well, so that you don't break anything?

Reply:
Sure :)